semeio/workflows/localisation/local_script_lib.py

```diff
@@ -744,6 +777,155 @@ def add_ministeps(
         ert_local_config.getUpdatestep().attachMinistep(ministep)
 
 
+def get_corr_group_spec(correlations_spec_list, name):
+    found = False
+    corr_spec = None
+    for count, corr_spec in enumerate(correlations_spec_list):
+        corr_group_name = corr_spec.name
```

Comment:
Dont need to store this to a variable, just do: `if corr_spec.name == name`

Reply:
Yes, fixed